internal/httphandler/utils.go

```diff
@@ -124,7 +80,7 @@ func extractVerifiedQueryPayload(w http.ResponseWriter, r *http.Request, queryTy
 	case constants.GetTxProof:
 		blockNum, txIndex, err := getBlockNumAndTxIndex(params)
```

Comment:
If in previous cases you replaced getSomething(params) function by new httputils.GetSomething(params), this one should go there as well...

Reply:
I only moved to the common `httputils` package the minimal set of functions I needed for the catch-up service... (i.e. what's coming in the next PR). I think it is better to keep them here for now,, and if I'll need them there, I'll move them at that time. 
What do you think?

Reply:
Done